packages/phoenix-evals/src/phoenix/evals/models/anthropic.py

```diff
@@ -39,6 +39,8 @@ class AnthropicModel(BaseModel):
             body (e.g., countPenalty for a21 models). Defaults to an empty dictionary.
         max_content_size (Optional[int], optional): If using a fine-tuned model, set this to the
             maximum content size. Defaults to None.
+        initial_rate_limit (int, optional): The initial internal rate limit for making LLM calls.
+            This limit adjusts dynamically based on rate limit errors. Defaults to 5.
```

Comment:
It's not clear what the units are here. What is a 5 mean? Start at 5 concurrent requests?

Reply:
added units! This setting is in units of requests per second